### ISSUE TITLE
[EPIC] LAC tsv extract for eCR

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/phdi#182]
### Why are we doing this?
As an LAC IT engineer, I need to pass tsv eCR data that matches the IRIS required fields to LAC’s import utilities to seed IRIS and make that data available to case managers. 
For this to happen, we need a way to automatically provide a tsv with all the required eCR fields after an eCR has been processed through the pipeline.

(Gathering those fields and generating an initial schema is a separate ticket - this epic is only intended to track the work needed to generate these tsvs as a regular part of the pipeline.) 

### Action Requested
Broadly, we'll want to use the eCR database set up in #191 to populate these tsvs. Rather than putting the computational load on the FHIR server (and having to deal with the limits of FHIR server implementations), we can use the eCR database directly, and perform some kind of `select *` function for tsv population.

### Acceptance Criteria
After an eCR travels through the LAC implementation of the pipeline, a tsv extract is made to populate their IRIS form. (Where these tsvs should live/be sent after they're created for LAC to process is TBD.)

**Comment by emmastephenson** [comment]
Related work: #191. The database will need to be in place before the tsv extract can be set up.
